src/views/Home/components/NewsCard/components/CategorizedNews/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Collapse from '@material-ui/core/Collapse';
+import { Button } from '@material-ui/core';
+import { gordonColors } from '../../../../../../theme';
+import NewsService from '../../../../../../services/news';
+import NewsItem from '../NewsItem';
+
+//Written by Jessica Guan
+
+export default class CategorizedNews extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleExpandClick = this.handleExpandClick.bind(this);
+
+    this.state = {
+      todaysNews: [],
+
+      open: false,
```

Comment:
comments explaining the states are very helpful when other begin to work on your code. todaysNews seems pretty self explanatory, however, I have not clue what open means.